.changeset/moody-numbers-move.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
```

Comment:
I chose patch because I'm just adding content that isn't being used yet, but I am modifying a widget's widgetExport object, so let me know if that means it should be a minor change.

Reply:
Updated!